python/lsst/pipe/tasks/isolatedStarAssociation.py

```diff
@@ -0,0 +1,595 @@
+#
+# LSST Data Management System
+# Copyright 2008-2021 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import numpy as np
+import pandas as pd
+from smatch.matcher import Matcher
```

Comment:
How is lsst-devl importing this in the stack? I can't find `smatch` in the lsst github repositories or the scipipe environment. I'm assuming that we must be shipping it somehow, I just don't understand how.